src/sdg/templates/producten/email/email_broken_links.html

```diff
@@ -0,0 +1,37 @@
+<div style="font-family: arial, sans-serif; font-size: 12px;">
+    <p>Beste {{ user_full_name }},</p>
+
+    <p>Je krijgt deze mail omdat jij een account hebt van de SDG-invoervoorziening.</p>
```

Comment:
```suggestion
    <p>Je krijgt deze mail omdat je een account hebt van de SDG-invoervoorziening en hebt aangegeven mails te willen ontvangen.</p>
```

Reply:
Fixed in https://github.com/maykinmedia/sdg-invoervoorziening/pull/1018/commits/adc06025a452bdece4d943240dc4dc302a80fc53 ✅